@here/harp-mapview/lib/MapView.ts

```diff
@@ -1828,6 +1839,8 @@ export class MapView extends THREE.EventDispatcher {
         }
         ++this.m_frameNumber;
 
+        this.m_animatedExtrusionHandler.zoom = this.m_zoomLevel;
```

Comment:
Too early.
`this.m_zoomLevel` for current frame is calculated in `this.updateCameras` call.

Current implementation always lags one render frame behind actual zoom level change.